tests/app/user/test_rest.py

```diff
@@ -1,8 +1,8 @@
 import json
+import unittest.mock as mock
```

Comment:
what happened with those imports? Did dependency bump mean they stopped working or something?

Reply:
Bumping the pytest dependencies meant the old mocks package was not being brought in anymore. 
We were already using unittest.mock elsewhere. From the mocks pypi: `mock is now part of the Python standard library, available as unittest.mock in Python 3.3 onwards.`